google-datacatalog-kafka-connector/README.md

```diff
@@ -126,17 +126,47 @@ export KAFKA2DC_KAFKA_HOST=kafka_bootstrap_server
 
 ```
 
+Export below variables to enable optional ingestion of the schemas from the Schema Registry. 
+The url is required, the rest depends on your Schema Registry authentication setup.
+
+```bash
+# Required to connect to the Schema Registry
```

Comment:
We need a description saying what is a schema registry (and link to confluent site?) together with a longer description what is scraped from there and that it is purely optional

Reply:
Done